lib/dynamoid/persistence.rb

```diff
@@ -134,6 +140,30 @@ def dynamo_type(type)
         end
       end
 
+      private
+
+            def transform_hash(hash)
```

Comment:
Also, transform doesn't describe which direction the translation is happening. I have to look at where it's called to understand that this is during the *deserializing* process.